### ISSUE TITLE
content_object resolves to None (BaseObjectPermission)

### ISSUE THREAD

**Issue by ZuSe** [django-guardian/django-guardian#748]
Hi guys,

we are seeing very weird behavior on our system that we can't explain.

If check the attached log you see that self.content_object is resolved as None.
However the toString of the statement before clearly shows that it is None but probably set as part of the permission.

The bug is only occurring on our kubernetes cluster with multiple nodes. We have honestly no idea what could cause it. 
After debugging for several hours I open the ticket now in the hope that someone else ran into it. I will keep you updated as soon as find out sth.

Environment:
- Python 3.9.5
- gunicorn 20.1.0 with uvicorn 0.13.4 (asgi3)
- Postgres 13.3
- Django 3.2

![image](https://user-images.githubusercontent.com/522403/119153258-62583f80-ba51-11eb-8fd6-8a416c45d913.png)

**Comment by misoomang** [comment]
I have the same problem. Using pdb to debug, I found self.content_object was None at begining, after a moment, it's not None

<img width="657" alt="wecom-temp-d36ce051133a8394d2860255e61b360c" src="https://user-images.githubusercontent.com/24948343/156193281-eaa74627-7811-40df-b9fb-cf3e1ac90142.png">

**Comment by kmmbvnr** [comment]
Can it be the case that the referred objects was deleted?

https://github.com/django/django/commit/1a82fc245eb8ac4b131ec02a6ac3e112deb8d5a6

**Comment by thclark** [comment]
I agree with @kmmbvnr - running clean_orphan_obj_perms might help you out too.
